idea/testData/codeInsight/surroundWith/if/moveDeclarationsOut/val/fullQualifiedType.kt.after

```diff
@@ -1,7 +1,6 @@
 fun foo() {
-    val a: kotlin.test.Asserter?
-    if (<caret>) {
-        a = null
+    val a = if (<caret>) {
```

Comment:
Right. I think if there is type declaration, this intention should keep them.

```kotlin
val a: kotlin.test.Asserter? = if (<caret>) {
    null
}
```

should be the expected result. I will work on that.